xpk.py

```diff
@@ -1020,6 +1020,24 @@ def set_jobset_on_cluster(args) -> int:
         'jobset command on server side returned with ERROR returncode'
         f' {return_code}.\n'
     )
+    xpk_print(
+      'This likely means you\'re missing Kubernetes Permissions, you can'
```

Comment:
Seems super duplicative. Sure you don't want to link to the README and have a deeplink in the README?

Reply:
Agreed and added a link to the url weblink to the readme in the error message.